src/modules/75-cd/pages/get-started-with-cd/CDOnboardingWizardWithCLI/Steps/VerificationComponents/VerifyGitopsEntities.tsx

```diff
@@ -0,0 +1,199 @@
+import React, { useEffect, useMemo } from 'react'
+import { useParams } from 'react-router-dom'
+import { Button, ButtonVariation, IconName, IconProps, Layout, Text } from '@harness/uicore'
+import { Color, FontVariation } from '@harness/design-system'
+import { ProjectPathProps } from '@common/interfaces/RouteInterfaces'
+import { useTelemetry } from '@common/hooks/useTelemetry'
+import { String, useStrings } from 'framework/strings'
+import { useAgentRepositoryServiceGet, useAgentApplicationServiceGet, useAgentClusterServiceGet } from 'services/gitops'
+import SuccessBanner from './SuccessBanner'
+import { useOnboardingStore } from '../../Store/OnboardingStore'
+import { CDOnboardingSteps, PipelineSetupState, WhatToDeployType, WhereAndHowToDeployType } from '../../types'
+import { GITOPS_ENTITY_IDS_BY_DEPLOYMENT_TYPE } from '../../Constants'
+import { ONBOARDING_INTERACTIONS } from '../../TrackingConstants'
+import { getBranchingProps } from '../../utils'
+import css from '../../CDOnboardingWizardWithCLI.module.scss'
+
+export default function VerifyGitopsEntities({
+  saveProgress
+}: {
+  saveProgress: (stepId: string, data: unknown) => void
+}): JSX.Element {
+  const { getString } = useStrings()
+  const { stepsProgress } = useOnboardingStore()
+  const { trackEvent } = useTelemetry()
+  const { accountId, orgIdentifier, projectIdentifier } = useParams<ProjectPathProps>()
+  const configdata = stepsProgress?.[CDOnboardingSteps.HOW_N_WHERE_TO_DEPLOY]?.stepData as WhereAndHowToDeployType
+  const entityIds = useMemo((): { application: string; cluster: string; repo: string } => {
```

Comment:
If you're trying to mark the expected output of the `useMemo`, you can do that by passing the generic.
```suggestion
  const entityIds = useMemo<{ application: string, cluster: string, repo: string }>(() => {
```